src/Symfony/Bundle/FrameworkBundle/Tests/Command/CacheClearCommand/CacheClearCommandTest.php

```diff
@@ -38,7 +38,7 @@ protected function setUp(): void
     protected function tearDown(): void
     {
         $this->fs->remove($this->kernel->getProjectDir());
-        $this->fs->remove(__DIR__.'/Fixture/preload.php');
+        $this->fs->remove(__DIR__.'/Fixture/.preload.php');
```

Comment:
But the file is renamed to /preload.php, isn't it? 🧐

Reply:
it was rename from preload.php to .preload.php (to exclude it from the container list of managed files)